_posts/2012-01-19-format-efficiency-take-2-really-clean-json.html

```diff
@@ -112,30 +112,30 @@ <h2>Approach</h2>
+<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>    &lt;</span></span><span><span style="color: #a31515">ValueAnnotation </span><span style="color: #e50000">Term</span><span style="color: #0000ff">=</span>"<span style="color: #0000ff">odata.urls.keylookupexpression</span>"<span style="color: #e50000"> Target</span><span style="color: #0000ff">=</span>"<span style="color: #0000ff">ODataDemo.DemoService.Products</span>"</span><span style="color: #0000ff">&gt;</span></span></p>
+<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>      &lt;</span></span><span><span style="color: #a31515">Apply </span><span style="color: #e50000">Function</span><span style="color: #0000ff">=</span>"<span style="color: #0000ff">KeyConcat</span>"</span><span style="color: #0000ff">&gt;</span></span></p>
 <p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>        &lt;</span></span><span><span style="color: #a31515">String</span><span style="color: #0000ff">&gt;</span>(<span style="color: #0000ff">&lt;/</span><span style="color: #a31515">String</span></span><span style="color: #0000ff">&gt;</span></span></p>
 <p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>        &lt;</span></span><span><span style="color: #a31515">Path</span><span style="color: #0000ff">&gt;</span>ID<span style="color: #0000ff">&lt;/</span><span style="color: #a31515">Path</span></span><span style="color: #0000ff">&gt;</span></span></p>
 <p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>        &lt;</span></span><span><span style="color: #a31515">String</span><span style="color: #0000ff">&gt;</span>)/<span style="color: #0000ff">&lt;/</span><span style="color: #a31515">String</span></span><span style="color: #0000ff">&gt;</span></span></p>
-<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>      &lt;/</span></span><span><span style="color: #a31515">Apply</span></span><span style="color: #0000ff">&gt;</span></span></p>
-<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>    &lt;/</span></span><span><span style="color: #a31515">ValueAnnotation</span></span><span style="color: #0000ff">&gt;</span></span><span style="font-family: Lucida Console"><span> </span></span></p>
+<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>      &lt;/</span></span><span><span style="color: #a31515">Apply </span></span><span style="color: #0000ff">&gt;</span></span></p>
+<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>    &lt;/</span></span><span><span style="color: #a31515">ValueAnnotation </span></span><span style="color: #0000ff">&gt;</span></span><span style="font-family: Lucida Console"><span> </span></span></p>
```

Comment:
```suggestion
<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>    &lt;/</span></span><span><span style="color: #a31515">ValueAnnotation</span></span><span style="color: #0000ff">&gt;</span></span><span style="font-family: Lucida Console"><span> </span></span></p>
```
This is a closing `</ValueAnnotation>` so we shouldn't have a space after `ValueAnnotation`